R/codecov.R

```diff
@@ -0,0 +1,143 @@
+#' Run covr on a package and upload the result to codecov.io
+#' @param path file path to the package
+#' @param Codecov url (change for Enterprise)
+#' @param ... additional arguments passed to \code{\link{package_coverage}}
+#' @export
+codecov <- function(path = ".", base_url = "https://codecov.io", ...) {
+  
+  # -------
+  # Jenkins
+  # -------
+  if (Sys.getenv("JENKINS_URL")) {
```

Comment:
`Sys.getenv("JENKINS_URL")` usually returns an empty string, which is not a valid logical expression to be passed to `if`. Maybe `Sys.getenv("JENKINS_URL") != ''`